emukit/bayesian_optimization/acquisitions/probability_of_improvement.py

```diff
@@ -55,9 +55,12 @@ def evaluate_with_gradients(self, x: np.ndarray) -> Tuple:
         dstandard_devidation_dx = dvariance_dx / (2 * standard_deviation)
 
         y_minimum = np.min(self.model.Y, axis=0)
-        pdf, cdf, u = get_quantiles(self.jitter, y_minimum, mean, standard_deviation)
 
-        dcdf_dx = - (pdf / standard_deviation) * (dmean_dx + dstandard_devidation_dx * u)
+        mean += self.jitter
+        u = (y_minimum - mean) / standard_deviation
```

Comment:
While `get_quantiles` probably raises a few design questions, removing it does leave the same code repeated 3 times. Maybe we should look into having a similar utility in emukit?

Reply:
It's in two places I think, both in `ExpectedImprovement`. 

The code in `ProbabilityOfImprovement` is slightly different as it gets the pdf of the normal N(x | mu, sigma) rather than N((x-mu)/sigma, 0, 1). 

I can add a helper function in the `ExpectedImprovement` file.

Reply:
fair enough